File: src/main/scala-graphql/io/github/graphql/scala/resolver/IssueContributionsByRepositoryResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._

/**
 * Resolver for IssueContributionsByRepository
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait IssueContributionsByRepositoryResolver {

    @javax.validation.constraints.NotNull
    @throws[Exception]
    def contributions(issueContributionsByRepositoryTO: IssueContributionsByRepositoryTO, after: String, before: String, first: Option[Int], last: Option[Int], orderBy: ContributionOrderTO): CreatedIssueContributionConnectionTO

}

File: src/main/kotlin-graphql/io/github/graphql/resolver/IssueContributionsByRepositoryResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for IssueContributionsByRepository
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
interface IssueContributionsByRepositoryResolver {

    @Throws(Exception::class)
    fun contributions(issueContributionsByRepositoryTO: IssueContributionsByRepositoryTO, after: String?, before: String?, first: Int?, last: Int?, orderBy: ContributionOrderTO?): CreatedIssueContributionConnectionTO

}
